packages/checkinstall.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Checkinstall < Package
+  description 'CheckInstall is a computer program for Unix-like operating systems which eases the installation and uninstallation of software compiled from source by making use of package management systems.'
+  homepage 'http://asic-linux.com.mx/~izto/checkinstall/'
+  version '1.6.2'
+  source_url 'http://asic-linux.com.mx/~izto/checkinstall/files/source/checkinstall-1.6.2.tar.gz'
+  source_sha256 'dc61192cf7b8286d42c44abae6cf594ee52eafc08bfad0bea9d434b73dd593f4'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  def self.patch
+    system 'wget', 'https://github.com/JL2210/patches/raw/f2da9fb13545311e67decef27626c45e4c6583e1/checkinstall-1.6.2-chromebrew.patch'
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read('checkinstall-1.6.2-chromebrew.patch') ) == '17706ab65698b7baa4cb2c1eda1c9ef3a41453069f6d83843f3c134439d3096a'
+    system 'patch -p1 < checkinstall-1.6.2-chromebrew.patch'
+  end
+
+  def self.build
+    system 'make'
+  end
+
+  def self.install
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/lib/checkinstall/"
```

Comment:
`#{CREW_DEST_PREFIX}/etc/checkinstall/` is better location IMHO.

Reply:
What does IMHO mean?

Reply:
Plus the patch doesn't allow for that.

Reply:
IMHO = in my humble opinion

Reply:
I'll change the patch. I see what you mean.